onnxscript/tests/function_libs/torch_aten/extra_opinfo.py

```diff
@@ -215,7 +216,36 @@ def sample_inputs_max_pool2d_with_indices(
         yield opinfo_core.SampleInput(arg, kwargs=kwargs)
 
 
+def sample_inputs_nn_col2im(op_info, device, dtype, requires_grad, **kwargs):
+    cases = (
+        # input_shape, output_size, kernal, dilation, padding, stride
+        ((1, 12, 12), (4, 5), (2, 2), 1, 0, 1),
+        ((1, 8, 30), (4, 5), (2, 2), 1, 1, 1),
+        ((1, 8, 9), (4, 4), (2, 2), 1, 0, 1),
+        ((1, 8, 25), (4, 4), (2, 2), 1, 1, 1),
+        ((1, 8, 9), (4, 4), (2, 2), 1, 1, 2),
+        ((1, 9, 4), (4, 4), (3, 3), 1, 1, 2),
+        ((1, 18, 16), (2, 2), (1, 1), 2, 3, 2),
+    )
+
+    make_arg = functools.partial(torch_testing.make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
+    for shape, output_size, kernel_size, dilation, padding, stride in cases:
+        tensor = make_arg(shape)
+        yield opinfo_core.SampleInput(tensor, output_size, kernel_size, dilation, padding, stride)
+
+
 OP_DB: List[opinfo_core.OpInfo] = [
+    opinfo_core.OpInfo('nn.functional.fold',  # This name must be a valid function in torch
```

Comment:
Is this right? I am pretty sure the name can be anything?

Reply:
If change to nn.functional.col2im will get a failed.

Reply:
![error](https://user-images.githubusercontent.com/19218188/230694437-28250858-6121-482d-bb26-83bd80a664f1.png)

Reply:
Let me try something

Reply:
I just renamed to col2im. I will see if it works and will revert if it doesn't

Reply:
Fixed

Reply:
The trick is to populate the `op` field of `OpInfo`